resources/templates/md/pages/user-guide.md

```diff
@@ -783,18 +783,36 @@ View the leaderboard:
 !karma
 ```
 
-Give Yetibot some ❤️:
+There are three ways to adjust a user's Karma.
+
+The canonical `!karma` command invocation, using the traditional C increment (`++`) or decrement (`--`) operators, optionally followed by a note.
 
 ```yetibot
 !karma @yetibot++ thanks Yetibot
 ```
 
-Give Yetibot some 💔:
-
 ```yetibot
 !karma @yetibot-- you had one job
 ```
 
+The Emoji versions to add (🌈) or subtract (⛈), optionally followed by a note.
+
+```plain
+🌈 @yetibot 
+```
+
+```plain
+⛈ @yetibot I told you to go before we got in the car!
+```
+
+Via Emoji responses, which necessarily preclude the addition of notes.
```

Comment:
Should we reference these as "Emoji reactions" since that's what [Slack calls them](https://get.slack.help/hc/en-us/articles/206870317-Emoji-reactions)?

Reply:
Yep, definitely.